src/api/app/helpers/webui/package_helper.rb

```diff
@@ -88,4 +88,10 @@ def binaries?(xml_hash)
   def repo_type_and_priority(repository)
     [repository.repo_type, repository.priority].compact.join(', Priority: ')
   end
+
+  def package_link_dest(pack, opts = {})
+    opts[:project] = User.current.branch_project_name(pack.project.name)
+    opts[:short] = true
+    project_or_package_link(opts)
```

Comment:
You will always link to a project, no? So we should use ``project_path`` (or ``link_to_project``) instead!

Reply:
Also if you use url_helper you can do it directly in the view and don't need to introduce a new helper!

However, if you stick to using a helper, it would be nice to add a helper test to
``src/api/spec/helpers/webui/package_helper_spec.rb`` (and a nice exercise as well :rofl: )

Reply:
`link_to_project` always creates a link, even when the project does not exist. `project_or_package_link` calls `link_to_project` if the project exists but does not if the project does not exists. I did not want to redo the checking of the project's existence.

I will have a look at `project_path` and either move it in the view or write a test.

Reply:
Or just do it in the view
```
project_or_package_link(User.current.branch_project_name(pack.project.name), short: true)
```